### DISCUSSION TITLE
Is there a way to do css type transform for a whole control for example a StackPanel?

### DISCUSSION THREAD

**Discussion by lanyusan** [AvaloniaUI/Avalonia#9529] [Q&A] [answered]
Our electron app uses css transform for zooming, which is simple and performant:

```
        transform: `scale(${scale})`,
        transformOrigin: 'top left',

```

Is there something similar with Avalonia?

**Comment by lanyusan** [comment, marked as answer]
It seems LayoutTransformControl is the way.

**Reply by maxkatz6** [reply]
Yes, if you want to change layout properties with a transform. So, layout elements will be rearranged for a new scaled size, for example.

But if you want to only scale rendering node, without affecting the layout, you need to use RenderTransform. I found this example, but that's with animations. Should be similar though https://docs.avaloniaui.net/docs/animations/transitions#transitioning-render-transforms
